.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -13,19 +13,16 @@ After:
 - [ ] **DONE**
 
 ## Documentation
-- No documentation needed: **add explanation. This can't be used if there is a GUI diff**
-- No documentation needed: only internal and user invisible changes
+- No documentation needed: **add explanation. This can't be used if there is a GUI diff**, one can be "only internal and user invisible cnages"
 - Documentation issue was created: [Link for SUSE Manager contributors](https://github.com/SUSE/spacewalk/issues/new?template=ISSUE_TEMPLATE_DOCUMENTATION.md&labels=documentation&projects=SUSE/spacewalk/31), [Link for community contributors](https://github.com/uyuni-project/uyuni-docs/issues/new).
 - API documentation added: please review the Wiki page [Writing Documentation for the API](https://github.com/uyuni-project/uyuni/wiki/Writing-documentation-for-the-API) if you have any changes to API documentation.
 - (OPTIONAL) [Documentation PR](https://github.com/uyuni-project/uyuni-docs/pulls)
 
 - [ ] **DONE**
 
 ## Test coverage
-- No tests: **add explanation**
-- No tests: already covered
-- Unit tests were added
-- Cucumber tests were added
+- No tests: **add explanation** (one explanation can be "already covered")
+- Tests were added: **add explanation**
```

Comment:
Do we need an explanation for added tests?

Reply:
Requesting new review.